app/Http/Controllers/UserController.php

```diff
@@ -28,7 +28,7 @@ public function __construct(RepositorieFactory $repositoryFactory)
     {
         $this->middleware('auth');
         //in the update en edit methode we check if the user is editing his own page or he is a administrator
-        $this->middleware('authorize:'. Config::get('constants.Administrator'))->except(['edit','update','show']);
+        $this->middleware('authorize:'.\Config::get('constants.Administrator') .',' . \Config::get('constants.Certificate_administrator'))->except(['edit','update','show']);
```

Comment:
Ik denk dat Certificate Administrators nu wel nieuwe users toe kunnen voegen, en alle users kunnen exporteren. Als dit geen probleem is, ziet het er voor de rest prima uit.